tox.ini

```diff
@@ -50,7 +50,7 @@ deps =
 
 commands =
     black --check --diff --target-version py38 src tests setup.py
-    isort --check-only --diff -rc tests src setup.py
+    isort --check-only --profile black --diff -rc tests src setup.py
```

Comment:
Helps linting conflicts that can occur between isort and black [link](https://github.com/PyCQA/isort/issues/1518)